ui/admin/tests/e2e/README.md

```diff
@@ -0,0 +1,44 @@
+# Admin UI e2e tests
+
+This test suite tests the Boundary Admin UI in an end-to-end setting. It was designed to be run in a
+variety of environments as long as the appropriate environment variables are set. You can choose to
+spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
+to generate it for you. There is an enos scenario in the `boundary` repository that runs these
+tests.
+
+The test suite uses the [Playwright](https://playwright.dev/) framework.
+
+## Getting Started
+Setup...
+```bash
+cd boundary-ui/ui/admin
+yarn install
+npx playwright install
```

Comment:
What is the purpose of this command?  Can it be included in the package.json as a `pre:e2e` step?  One less thing for developers to worry about.

Reply:
It's a one-time thing to install the browsers used by Playwright. Does the `pre:` imply something/is that a special command?

Reply:
My understanding of `npx playwright install` as Michael mention is an optional command to pre-install all browsers (one time) within your local machine before running the test. If you do not use this command is totally fine, browsers will be install (just one time) when developer runs the test.

I will suggest (for now) to not add any yarn script to run `npx playwright install`. I will leave it on the documentation as optional to keep it simple until we anticipate we have this need.

@randallmorey Happy to have a char over the `npx playwright install` detailed script if you want

Reply:
@moduli @randallmorey 
You guys are right. Is best to have the `npx playwright install` and also a `pre:` script that takes care of it.

I checked and if you already have the browsers install, running `npx playwright install` does not try to resintall them again, so is better to guarantee test suite integrity